cpp/log/cluster_state_controller.h

```diff
@@ -61,6 +61,12 @@ class ClusterStateController {
 
   void RefreshNodeState();
 
+  bool NodeIsStale() const;
+
+  // Returns a vector of nodes in the cluster which are able to serve the
+  // cluster's current ServingSTH.
```

Comment:
This always exclude the current node, even if it *is* funky fresh, maybe mention that here?